assets/js/modules/optimize/components/common/AMPExperimentJSONField.js

```diff
@@ -61,16 +65,22 @@ export default function AMPExperimentJSONField() {
 	return (
 		<Fragment>
 			<p>
-				{ __(
-					'Please input your AMP experiment settings in JSON format below.',
-					'google-site-kit'
-				) }{ ' ' }
-				<Link
-					href="https://developers.google.com/optimize/devguides/amp-experiments"
-					external
-				>
-					{ __( 'Learn more', 'google-site-kit' ) }
-				</Link>
+				{ createInterpolateElement(
+					__(
+						'Please input your AMP experiment settings in JSON format below.',
```

Comment:
```suggestion
						'Please input your AMP experiment settings in JSON format below. <LearnMoreLink />',
```

Or even like in previous suggestions to use `... <a>Learn more</a>`.